pkg/registry/rbac/clusterrolebinding/policybased/storage.go

```diff
@@ -59,7 +60,7 @@ func (s *Storage) Create(ctx context.Context, obj runtime.Object, createValidati
 		return s.StandardStorage.Create(ctx, obj, createValidation, includeUninitialized)
 	}
 
-	rules, err := s.ruleResolver.GetRoleReferenceRules(clusterRoleBinding.RoleRef, metav1.NamespaceNone)
+	rules, err := s.ruleResolver.GetRoleReferenceRules(rbacv1helpers.ConvertOrDieRoleRef(clusterRoleBinding.RoleRef), metav1.NamespaceNone)
```

Comment:
`___OrDie` inside a rest handler isn't something I'd expect

Reply:
> ___OrDie inside a rest handler isn't something I'd expect

I'm willing to change it, but it is a little difficult to come up with a way it won't round trip.